### DISCUSSION TITLE
ActiveJ in Scala ?

### DISCUSSION THREAD

**Discussion by anboo44** [activej/activej#307] [Q&A] [answered]
Can you make activeJ in Scala ?

**Comment by eduard-vasinskyi** [comment, marked as answer]
Hi, @anboo44

You can use ActiveJ in Scala. There was a recent [discussion](https://github.com/activej/activej/discussions/302) about ActiveJ + Scala with ActiveJ serving as an application launcher and HTTP server. There is a link to the GitHub repository of the project, you could use that as a reference.
